### PR TITLE
Add support for querying Receipts

### PR DESCRIPTION
**Pull request by Salka1988**
I've generated new types on the `fuel-graph-node` side and extended the `fuel.ts` file on the `fuel-graph-tooling` side.

### CODE REVIEW

File: firehose-fuel/devel/fuelfire/fuelfire.yaml [hunk 1]
```diff
@@ -6,8 +6,11 @@ start:
     - relayer
   flags:
     # Specifies the path to the fuel-node binary.
-    reader-node-path: "./../../../../fullstack/fuel-core-with-firehose-node/fuel-core/target/debug/fuel-core"
+    reader-node-path: "./../../../firehose-extract/target/debug/firehose-extract"
```
Comment (Dentosal) [inline firehose-fuel/devel/fuelfire/fuelfire.yaml:9]: We should probably use release build here

File: firehose-fuel/proto/sf/fuel/type/v1/type.pb.go
Comment (Dentosal) [inline firehose-fuel/proto/sf/fuel/type/v1/type.pb.go:1]: Deduplicated? Good.

File: fuel-graph-node/chain/fuel/src/protobuf/sf.fuel.r#type.v1.rs
Comment (Dentosal) [inline fuel-graph-node/chain/fuel/src/protobuf/sf.fuel.r#type.v1.rs:1]: This looks like generated Rust code that likely should not be committes. Use build.rs instead.

File: fuel-graph-node/graph/src/runtime/mod.rs [hunk 2]
```diff
@@ -392,6 +392,22 @@ pub enum IndexForAscTypeId {
     FuelStorageSlotArray = 5420,
     FuelBytesArray = 5421,
     FuelU64Array = 5422,
+    FuelReceipt = 5423,
+    FuelReceiptArray = 5424,
```
Comment (Dentosal) [inline fuel-graph-node/graph/src/runtime/mod.rs:396]: Looks like you apply the same change to multiple enums. Maybe we could import the other enum instead?

File: fuel-graph-tooling/packages/ts/chain/fuel.ts
Comment (Dentosal) [inline fuel-graph-tooling/packages/ts/chain/fuel.ts:1]: The formatting should be enforced in the ci

**Review by Dentosal — CHANGES_REQUESTED** [review]

File: fuel-graph-node/graph/src/runtime/mod.rs (hunk 2 above)
Comment (Salka1988) [reply fuel-graph-node/graph/src/runtime/mod.rs:396]: I think we have to do like this here and on the .ts side.

File: fuel-graph-tooling/packages/ts/chain/fuel.ts
Comment (Salka1988) [reply fuel-graph-tooling/packages/ts/chain/fuel.ts:1]: We should create a new issue for that.

File: firehose-fuel/devel/fuelfire/fuelfire.yaml (hunk 1 above)
Comment (Salka1988) [reply firehose-fuel/devel/fuelfire/fuelfire.yaml:9]: We can include this in the issue where we will add ci.yml ?

**Review by Dentosal — APPROVED** [review]
Looks good. Generates some follow-up issues, but seems generally fine.

File: firehose-fuel/devel/fuelfire/fuelfire.yaml (hunk 1 above)
Comment (Salka1988) [reply firehose-fuel/devel/fuelfire/fuelfire.yaml:9]: Here it is: https://github.com/FuelLabs/fuel-subgraph/issues/12

**Pull request merged**